src/helpers/assign.js

```diff
@@ -1,3 +0,0 @@
-/* global Sandpit */
-Object.assign(Sandpit.default, Sandpit)
```

Comment:
That's a good question - is there a way to achieve what it was achieving with rollup? I find having to write ```new Sandpit.default('body', Sandpit.default.CANVAS)``` a bit clunky?